CHANGELOG.md

```diff
@@ -3,9 +3,12 @@
 `servant-hmac-auth` uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
 
-## 0.1.4 - Jan 27, 2023
+## 0.1.5 - Jan 27, 2023
 * Bump dependency upper bounds, allow building with `GHC 9.0`, `9.2` and `9.4`
 
+## 0.1.4 - March 8, 2022
+* [#55](https://github.com/Holmusk/servant-hmac-auth/pull/55):
```

Comment:
```suggestion
* [#55](https://github.com/Holmusk/servant-hmac-auth/pull/55): **Breaking change**: non-standard http(s) port are now included in the HMAC signature. _Migration guide_: if you are communicating over the standard http(s) port (`80` for http, `443` for https), then you will **not** be impacted. Otherwise, you need to make sure that both the client and server include the custom http(s) port number in the HMAC signature generation. This library will do so automatically.
```